ia/src/ml_agent.py

```
@@ -20,79 +21,86 @@ def get_position(index, total_cells) -> tuple[int, int]:
+            if resource_name in cell.lower():
+                x, y = get_position(index, len(vision_data))
+                real_distance = math.sqrt(x*x + y*y)
+                
+                if real_distance < min_distance:
+                    min_distance = real_distance
+                    closest_resource = {
+                        'x': x,
+                        'y': y,
+                        'real_distance': real_distance,
+                        'resource_name': resource_name,
+                        'content': cell
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)